src/main/java/cceclipseplugin/core/EclipseRequestManager.java

```diff
@@ -95,6 +95,8 @@ public void finishSubscribeToProject(long id, File[] files) {
 				System.err.println(String.format("File %s was pulled with version 0.", f.getFilename()));
 			}
 			fileMetadatas.add(new FileMetadata(f));
+			// note that this \/ path is wrong b/c it needs the project name on the front and then it needs to be normalized
+			// pm.putFileInWarnList(f.getRelativePath().toString() + "/" + f.getFilename(), FileChangeRequest.class);
```

Comment:
I mentioned that in the original PR post, basically it's not actually introduced by this PR, it already existed.